File: wallendar-app/app/src/main/java/com/td/wallendar/home/profile/ui/ProfileView.java
package com.td.wallendar.home.profile.ui;

import com.td.wallendar.home.profile.OnLogoutClickedListener;
import com.td.wallendar.home.profile.OnShowAliasesClickedListener;
import com.td.wallendar.models.ApplicationUser;

public interface ProfileView {
    void bind(ApplicationUser applicationUser,
              OnShowAliasesClickedListener onShowAliasesClickedListener,
              OnLogoutClickedListener onLogoutClickedListener);
}

File: wallendar-app/app/src/main/java/com/td/wallendar/home/profile/ui/ProfileView.kt
package com.td.wallendar.home.profile.ui

import com.td.wallendar.home.profile.OnLogoutClickedListener
import com.td.wallendar.home.profile.OnShowAliasesClickedListener
import com.td.wallendar.models.ApplicationUser

interface ProfileView {
    fun bind(
            applicationUser: ApplicationUser,
            onShowAliasesClickedListener: OnShowAliasesClickedListener,
            onLogoutClickedListener: OnLogoutClickedListener
    )
}